### ISSUE TITLE
Latest release

### ISSUE THREAD

**Issue by jmgirard** [bnosac/audio.whisper#61]
Should the latest release be bumped to 0.4? And should 0.4 be re-tagged to 0.4.0?

**Comment by jwijffels** [comment]
Yes, most likely, I´m doing some testing on some larger sets of audio files to see if I correctly implemented these offsets before I make a release

**Comment by jwijffels** [comment]
Closing as absolete due to upgrade https://github.com/bnosac/audio.whisper/releases/tag/0.5.0
